### ISSUE TITLE
orc-tools unknown subcommand "Merge"

### ISSUE THREAD

**Issue by echoeslove** [apache/orc#1920]
follow https://orc.apache.org/docs/java-tools.html

execute
```
java -jar orc-tools-2.0.0-uber.jar merge
```

display
```
Unknown subcommand: merge
```

**Comment by dongjoon-hyun** [comment]
Ya, thank you for reporting. 

It will be delivered via [ORC 2.0.1](https://github.com/apache/orc/milestone/29) in two weeks.

https://issues.apache.org/jira/browse/ORC-1644

**Comment by dongjoon-hyun** [comment]
This will be resolved via #1928 .

**Comment by cxzl25** [comment]
Now you can use 2.0.1 version.
```bash
wget https://repo1.maven.org/maven2/org/apache/orc/orc-tools/2.0.1/orc-tools-2.0.1-uber.jar
```
prepare data
```bash
echo -e "1,foo\n2,bar" > 1.csv
echo -e "3,apache\n4,orc" > 2.csv
java -jar orc-tools-2.0.1-uber.jar convert 1.csv --schema  'struct<c1:int,c2:string>'  --output 1.orc
java -jar orc-tools-2.0.1-uber.jar convert 2.csv --schema  'struct<c1:int,c2:string>'  --output 2.orc
````

use merge command
```bash
java -jar orc-tools-2.0.1-uber.jar merge --output merge.orc 1.orc 2.orc
```
```
Output path: merge.orc, Input files size: 2, Merge files size: 2
```

dump
```bash
java -jar orc-tools-2.0.1-uber.jar meta -d merge.orc
```
```
Processing data file merge.orc [length: 546]
{"c1":1,"c2":"foo"}
{"c1":2,"c2":"bar"}
{"c1":3,"c2":"apache"}
{"c1":4,"c2":"orc"}
```
